src/screens/AddNotificationEmailScreen.tsx

```diff
@@ -96,15 +99,16 @@ class AddNotificationEmailScreen extends PureComponent<Props, State> {
           return this.goToLocalEmailConfirm();
         }
         navigation.navigate(Route.ChooseWalletsForNotification, {
+          flowType: ConfirmAddressFlowType.SUBSCRIBE,
+          subtitle: i18n.notifications.getNotification,
+          description: i18n.notifications.chooseWalletsDescription,
           email,
           onSuccess,
-          walletsToSubscribe,
+          wallets: walletsToSubscribe,
           onSkip: () => this.goToLocalEmailConfirm(),
         });
       },
-      onFailure: (error: string) => {
-        this.setState({ error });
-      },
+      onFailure: () => this.setState({ localError: this.props.error }),
```

Comment:
`onFailure: (err) => this.setState({ localError: err })` or maybe like this so there will be only localError

Reply:
no, because we need here readable error from the selector, not from the store

Reply:
ah ok. Anyway I think this is not the best solution with this localError and error from redux (I know I did it by myself at the LocalNotificationConfirmScreen). The problem with this solution is that we have two sources of truth. Instead of setting localError there should be executed an action for setting error in redux. I'll refactor both it in the next PR

Reply:
I agree that's why I wanted to make id readable right in the saga so that we would always have only readable errors in store and could return it in onFailure

Reply:
readable errors shouldn't be added in sagas, selectors are the right place for mapping them

Reply:
yes, that's how it is